MarvelApiClient/CharactersApiClient.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CharactersApiClient.swift
+//  MarvelApiClient
+//
+//  Created by Pedro Vicente on 14/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BrightFutures
+
+public class CharactersApiClient : BaseApiClient {
```

Comment:
If we use getAll getById as method name, I prefer that we use the CharactersApiDatasource